do/loadbalancers.go

```diff
@@ -136,6 +138,8 @@ func (l *loadbalancers) GetLoadBalancer(clusterName string, service *v1.Service)
 //
 // EnsureLoadBalancer will not modify service or nodes.
 func (l *loadbalancers) EnsureLoadBalancer(clusterName string, service *v1.Service, nodes []*v1.Node) (*v1.LoadBalancerStatus, error) {
+	glog.Infof("ensuring load balancer cluster=%s service=%s", clusterName, service.Name)
```

Comment:
These logs could get pretty noisy since `EnsureLoadbalancer` can run during a resync. There's also some logging that happens in the controller https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/service/service_controller.go#L284-L310.